openedx/features/caliper_tracking/caliper_config.py

```diff
@@ -1,5 +1,6 @@
 from openedx.features.caliper_tracking.transformers.bookmark_transformers import edx_bookmark_listed
 from openedx.features.caliper_tracking.transformers.enrollment_transformers import edx_course_enrollment_activated
+from openedx.features.caliper_tracking.transformers.enrollment_transformers import edx_course_enrollment_mode_changed
```

Comment:
Don't add another line just to import another function from the same file.
import your function in the same import statement.

Reply:
Also, rebase the changes then make the PR.